packages/components/dialog/src/DialogCloseButton.tsx

```diff
@@ -0,0 +1,12 @@
+import * as RadixDialog from '@radix-ui/react-dialog'
+import { type ReactElement } from 'react'
+
+export type CloseButtonProps = RadixDialog.DialogCloseProps
+
+export const CloseButton = ({ children, ...rest }: CloseButtonProps): ReactElement => (
+  <RadixDialog.Close data-spark-component="dialog-close-button" {...rest}>
```

Comment:
Do we really want `data-spark-component` on each subcomponent from a compound one? Just asking, because it wasn't the case previously (see `Tabs`)

Reply:
I'm not sure, it depends on what we want to do with our future Chrome extension. I think we at least need to be able to visually detect the Trigger VS the Popover.Content. I can remove the reste maybe.

Reply:
There is no problem there, but I guess we should all have an agreement on this, to keep our strategy consistent :+1: